### DISCUSSION TITLE
Hybrid map is blury sometimes

### DISCUSSION THREAD

**Discussion by Sanookmakmak** [photoprism/photoprism#2631] [Q&A]
Please compare that satellite view https://benmaps.fr/@99.933650,9.537006,17.05 with that of PhotoPrism.
Both are using Openstreetmap but the map of PhotoPrism is blury.

![grafik](https://user-images.githubusercontent.com/24833757/185819548-b313149b-b860-4b8e-9a98-a94897db9c39.png)

**Comment by lastzero** [comment]
How can you enable satellite maps with the link you provided? I only see a text input at the top. Note satellite images are NOT part of OpenStreetMap but provided by independent vendors (the project does not operate any satellites).
